samples/MauiEmbedding/MauiEmbedding.MauiControls/SyncfusionDemoPage.xaml.cs

```diff
@@ -5,5 +5,15 @@ public partial class SyncfusionDemoPage : ContentPage
 	public SyncfusionDemoPage()
 	{
 		InitializeComponent();
+
+		Loaded += SyncfusionDemoPage_Loaded;
+	}
+
+	private void SyncfusionDemoPage_Loaded(object sender, EventArgs e)
+	{
+		var pv = Window?.Handler?.PlatformView;
+		var hpv = Handler?.PlatformView;
+
+		var current = DeviceDisplay.Current;
 	}
```

Comment:
These are variable declarations without any logic that does something. Is that intended?

Reply:
yes, just there for debugging - we'll clean this up once we finalise the solution for loaded event etc